### ISSUE TITLE
Guidance on saving large-ish files (6MB)?

### ISSUE THREAD

**Issue by kaxline** [etesync/etebase-js#41]
I'm getting a `413 Payload Too Large` error when trying to create an `Item` with binary content that's 6MB. I haven't tested where the cutoff is yet, but I was hoping there was some general advice on how to handle large files. 

I need to be able to handle single files that are up to 1GB or so of binary data.

Should I zip the file up and break it into separate `Items`? Or is there some way to handle this with Etebase that I'm not seeing/understanding.

Thanks!

**Comment by tasn** [comment]
Is this on your own server or on api.etebase.com?

**Comment by kaxline** [comment]
@tasn Sorry, I should've indicated that. It's on `api.etebase.com`. I can switch over to self-hosting if that's the issue.

**Comment by tasn** [comment]
Nah, I was going to say that I believe that etebase should defo handle it and maybe it was a misconfiguration in your self-hosted instance.

Etebase should (unless I'm misremembering) automatically split files into smaller chunks. What version of the lib are you using?

**Comment by kaxline** [comment]
@tasn I just did a fresh install of `0.43.1` and it's still happening. I can look into it, but I was just wondering if you knew off the top of your head whether that file size is inherently problematic or not.

**Comment by tasn** [comment]
I don't think it is. From the top of my head we automatically split to chunks, and anyway, even a chunk sized 6mb is fine.
